python/samples/getting_started/azure_functions/03_reliable_streaming/README.md

```diff
@@ -0,0 +1,438 @@
+    "AzureWebJobsStorage": "UseDevelopmentStorage=true"
+  }
+}
+```
+
+Configuration options:
+- `REDIS_CONNECTION_STRING`: Connection string for Redis (default: `redis://localhost:6379`)
+- `REDIS_STREAM_TTL_MINUTES`: Time-to-live for stream entries in minutes (default: `10`)
+
+## Running the Sample
+
+1. **Start required services** (Redis, DTS, Azurite):
+   ```bash
+   # Redis (if not already running)
+   docker run -d --name redis -p 6379:6379 redis:latest
+
+   # DTS and Azurite (see parent README for instructions)
+   ```
+
+2. **Install dependencies**:
+   ```bash
+   cd python/samples/getting_started/azure_functions/03_reliable_streaming
+   pip install -r requirements.txt
+   ```
+
+3. **Start the Function App**:
+   ```bash
+   func start
+   ```
+
+## Testing the Sample
+
+### Test 1: Basic Workflow (Standard + Custom Endpoints)
+
+**Step 1:** Start the agent run:
+
+```bash
+curl -X POST http://localhost:7071/api/agents/TravelPlanner/run \
+  -H "Content-Type: text/plain" \
+  -d "Plan a 3-day trip to Tokyo"
+```
+
+**Response (202 Accepted):**
+```json
+{
+  "status": "accepted",
+  "response": "Agent request accepted",
+  "message": "Plan a 3-day trip to Tokyo",
+  "conversation_id": "abc-123-def-456",
+  "correlation_id": "xyz-789-ghi-012"
+}
+```
+
+**Step 2:** Stream chunks from Redis using the custom endpoint:
+
+```bash
+curl http://localhost:7071/api/agent/stream/abc-123-def-456 \
+  -H "Accept: text/event-stream"
+```
+
+**Expected Response (SSE format with chunks):**
+```
+id: 1734649123456-0
+event: message
+data: Here's a wonderful 3-day Tokyo itinerary...
+
+id: 1734649123789-0
+event: message
+data: Day 1: Arrival and Shibuya...
+
+id: 1734649124012-0
+event: done
+data: [DONE]
+```
+
+**Step 3 (Optional):** Resume from a specific cursor:
+
+```bash
+# Use a cursor from an earlier SSE event
+curl "http://localhost:7071/api/agent/stream/abc-123-def-456?cursor=1734649123456-0" \
+  -H "Accept: text/event-stream"
+```
+
+### Test 2: Plain Text Format (for terminals)
+
+```bash
+# Start the run
+RESPONSE=$(curl -s -X POST http://localhost:7071/api/agents/TravelPlanner/run \
+  -H "Content-Type: text/plain" \
+  -d "Plan a weekend in Paris")
+
+CONV_ID=$(echo $RESPONSE | jq -r .conversation_id)
+
+# Stream in plain text format
+curl http://localhost:7071/api/agent/stream/$CONV_ID \
+  -H "Accept: text/plain"
+```
+
+### Test 3: Reading from Redis using redis-cli
+
+```bash
+# Connect to Redis
+docker exec -it redis redis-cli
+
+# View all messages for the conversation
+XRANGE agent-stream:abc-123-def-456 - +
+
+# Example output:
+# 1) 1) "1734649123456-0"
+#    2) 1) "text"
+#       2) "Here's a wonderful 3-day Tokyo itinerary..."
+#       3) "sequence"
+#       4) "0"
+#       5) "timestamp"
+#       6) "1734649123456"
+# 2) 1) "1734649123789-0"
+#    2) 1) "text"
+#       2) "Day 1: Arrival and Shibuya..."
+#       3) "sequence"
+#       4) "1"
+```
+
+### Test 4: Check Agent Status
```

Comment:
Other readme's are smaller and more concise, we should probably keep this consistent with the others too and remove all extra info.